app/scripts/directives/resources.js

```diff
@@ -165,9 +165,46 @@ angular.module('openshiftConsole')
       restrict: 'E',
       scope: {
         pods: '=',
+        // Optional active pods map to display whether or not pods have endpoints
+        activePods: '=?',
         // Optional empty message to display when there are no pods.
-        emptyMessage: '=?'
+        emptyMessage: '=?',
+        // Alternative header text to display in the 'Name' column.
+        customNameHeader: '=?',
+        // Optional map of explanations or warnings for each phase of a pod
+        podFailureReasons: '=?'
       },
       templateUrl: 'views/directives/pods-table.html'
     };
+  })
+  .directive('routesTable', function() {
```

Comment:
you aren't actually using the routesTable anywhere except the routes page right? i would just remove the directive and revert routes.html